skipper-py/src/state.py

```diff
@@ -50,6 +52,20 @@ async def set_all_pool_contracts(self,
                         router_contracts=router_contracts
                         )
         
+        # remove blacklisted pools
+        for address in list(self.contracts.keys()):
+            if address in self.blacklisted_contracts:
+                self.contracts.pop(address)
+        
+        if precheck:
+            for addr, pool in list(self.contracts.items()):
+                try:
+                    await pool.update_tokens(querier)
+
+                except Exception as e:
+                    print("pop " + addr)
```

Comment:
Can we log failures here?

Reply:
Also what are the possible failure cases for this pre-check? Could there be rpc-unreliability, etc.? I'd be interested in knowing what these are to avoid false-positives

Reply:
In the case of Astroport some pool contracts returned from the router cannot be queried and return this error instead ```Error: Query failed with (6): Error calling the VM: Error executing Wasm: Wasmer runtime error: RuntimeError: Error calling into the VM's backend: Panic in FFI call: query wasm contract failed: unknown request
```

I include those into the BLACKLIST_CONTRACTS env variable. This precheck is by default turned off, I used it to identify these non-working pools